community/lld/build

```diff
@@ -2,6 +2,9 @@
 
 export DESTDIR="$1"
 
+mkdir -p include/mach-o
+mv libunwind/include/mach-o/compact_unwind_encoding.h include/mach-o/compact_unwind_encoding.h
```

Comment:
Could be `install -Dm644 libunwind/include/mach-o/compact_unwind_encoding.h include/mach-o/compact_unwind_encoding.h`

Reply:
Happy to change that now!